django/exercises/migrations/0002_data_initial.py

```diff
@@ -143,8 +143,8 @@ def insert_themes(apps, schema_editor):
     """
 
     themes = [
-        "Test theme 1",
-        "Test theme 2"
+        "Grammer",
```

Comment:
Is the theme actually named 'Grammer'? Or is this a typo (Grammar)?

Reply:
it's a (somewhat ironic) typo - thanks for spotting. have fixed this.